app/jobs/update_sample_metadata_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class UpdateSampleMetadataJob < ApplicationJob
+  queue_with_priority 1
+  queue_as QueueNames::SAMPLES
+
+  def perform(sample_type, attribute_changes = [], user)
+    Seek::Samples::SampleMetadataUpdater.new(sample_type, attribute_changes, user).update_metadata
```

Comment:
General question, but what happens if someone views a sample before the job has finished running?

Reply:
Is there a way to test this?

Reply:
On your local instance you could update a sample type without any workers running. Alternatively, add a `sleep` here or in the job.